### DISCUSSION TITLE
Removed from testflight?

### DISCUSSION THREAD

**Discussion by Naozumi520** [ppy/osu#22396] [Q&A] [answered]
Just opened my testflight and saw this message:
<img src="https://cdn.discordapp.com/attachments/944037889375875112/1067438764726698064/IMG_0088.png" height="500" />

**Comment by mk56-spn** [comment]
Inactive users are periodically purged from testflights to make room for incoming people, so that's probably the most likely cause.

 Did you not play for an extended period of time prior to this happening by any chance?

**Reply by Naozumi520** [reply]
Yeah I saw the message from PPY on twitter about inactive users will getting removed.
But I have always installed the latest version, so this should not be the case here.
Also, it's only been two weeks since the last time osu lazer was opened.

Here is the proof:
<img src="https://cdn.discordapp.com/attachments/939482444778864663/1067459780739878952/IMG_0725.png" height="500" />

**Comment by Naozumi520** [comment, marked as answer]
https://twitter.com/ppy/status/1618457416756256769?cxt=HHwWgoDTncKc9fUsAAAA
